src/Gyselroth/Helper/HelperArray.php

```diff
@@ -1071,4 +1071,34 @@ public static function sortElementArr(array $elements)
     {
         return self::sortElements($elements);
     }
+
+    /**
+     * Reform items of given key(s) of all sub-arrays (1 level of depth) of given array to given type
+     *
+     * @param  array        $array
+     * @param  array|string $keys Multiple keys as array | One key as string
+     * @param  string       $type
+     * @return array
+     */
+    public static function castValSubItemsByKey(array $array, $keys, string $type = 'int'): array
```

Comment:
please improve (shorten, clarify, follow php-standards) the signature (method and argument naming)
-avoid unnecessary abbreviations
-you cannot cast anything other than values

-> how about: castSubColumn(array $array, $column, string $type) ?

p.s. following php standards and as the argument already is of "mixed" type- how about also allowing int (=index) for $column?